mpconstants/carriers.py

```diff
@@ -119,4 +119,312 @@
     }
 }
 
```

Comment:
I wonder if these should reference a variable instead of typing the same string over and over?